docs/src/user_docs/stdlib/index.md

```diff
@@ -32,7 +32,7 @@ Currently, Miden standard library contains just a few modules, which are listed
 | Module                                                              | Description                                                                                                                                                      |
 |---------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | [std::collections::mmr](./collections.md#merkle-mountain-range)     | Contains procedures for manipulating [Merkle Mountain Ranges](https://github.com/opentimestamps/opentimestamps-server/blob/master/doc/merkle-mountain-range.md). |
-| [std::crypto::fri::frie2f4](./crypto/fri.md#fri-extension-2-fold-4) | Contains procedures for verifying FRI proofs (field extension = 2, folding factor = 4).                                                                          |
+| [std::pcs::fri::frie2f4](./pcs/fri.md#fri-extension-2-fold-4) | Contains procedures for verifying FRI proofs (field extension = 2, folding factor = 4).                                                                          |
```

Comment:
I noticed that this list is missing:

- `stark` module.
- `sys::vm` module.

Could we add these? This would also require corresponding module doc files.